### PR TITLE
(maint) Update branding urls

### PR DESCRIPTION
**Pull request by jpogran**

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -23,6 +23,6 @@ PDK Version (if applicable):
 
 ### Output Log
 
-<!-- 
-For information how to capture verbose logs, look [here](https://github.com/lingua-pupuli/puppet-vscode/blob/master/client/README.md#2-capture-verbose-logs-and-send-them-to-us)
+<!--
+For information how to capture verbose logs, look [here](https://github.com/puppetlabs/puppet-vscode/blob/master/client/README.md#2-capture-verbose-logs-and-send-them-to-us)
```
Comment (DavidS) [inline .github/issue_template.md:27]:
```suggestion
For information how to capture verbose logs, look [here](https://github.com/puppetlabs/puppet-vscode/blob/master/README.md#2-capture-verbose-logs-and-send-them-to-us)
```

`/client` does not exist

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -3,11 +3,11 @@ BEFORE YOU CREATE A PULL REQUEST:
 
 Ensure you have read over
 
-CONTRIBUTING - https://github.com/lingua-pupuli/puppet-editor-services/blob/master/CONTRIBUTING.md
+CONTRIBUTING - https://github.com/puppetlabs/puppet-editor-services/blob/master/CONTRIBUTING.md
 
 and
 
-CODE_OF_CONDUCT - https://github.com/lingua-pupuli/puppet-editor-services/blob/master/CODE_OF_CONDUCT.md
+CODE_OF_CONDUCT - https://github.com/puppetlabs/puppet-editor-services/blob/master/CODE_OF_CONDUCT.md
```
Comment (DavidS) [inline .github/pull_request_template.md:10]: should this be replaced by https://github.com/puppetlabs/.github/blob/master/CODE_OF_CONDUCT.md ?

(there's also a CONTRIBUTING.md, but I'm not so sure about that one)

File: .github/issue_template.md (hunk 1 above)
Comment (jpogran) [reply .github/issue_template.md:27]: Good catch. This is a holdover from the first transition we did, back when client and server were in one repo!

File: .github/pull_request_template.md (hunk 2 above)
Comment (jpogran) [reply .github/pull_request_template.md:10]: Maybe, but should be part of another PR. Changing code of conduct and contributing is separate idea than changing links to the same content

File: .github/issue_template.md [hunk 3]
```diff
@@ -23,6 +23,6 @@ PDK Version (if applicable):
 
 ### Output Log
 
-<!-- 
-For information how to capture verbose logs, look [here](https://github.com/lingua-pupuli/puppet-vscode/blob/master/client/README.md#2-capture-verbose-logs-and-send-them-to-us)
+<!--
+For information how to capture verbose logs, look [here](https://github.com/puppetlabs/puppet-vscode/blob/master/README.md#2-capture-verbose-logs-and-send-them-to-us)
```
Comment (glennsarti) [inline .github/issue_template.md:27]: This should probably use our proper website?

https://puppet-vscode.github.io/docs/experience-a-problem/

File: .github/pull_request_template.md (hunk 2 above)
Comment (DavidS) [reply .github/pull_request_template.md:10]: yeah.

File: .github/issue_template.md (hunk 3 above)
Comment (jpogran) [reply .github/issue_template.md]: Updated

File: .github/pull_request_template.md (hunk 2 above)
Comment (jpogran) [reply .github/pull_request_template.md:10]: Created an issue to track the code of conduct and contributing updates. Resolving this

**Comment by glennsarti** [comment]
Ruh roh....I wonder if Facter 4 is breaking editor services :-(

**Pull request merged**